### ISSUE TITLE
remove md5

### ISSUE THREAD

**Issue by LeSpocky** [rafaelsteil/libcgi#15]
libcgi contains an implementation of the md5 hash function, however its not used inside the library itself and the status regarding license or potential security issues (despite the weekness of md5 itself) is not clear. If a user needs md5 he or she could get it from a specialized library.

(Comments welcome.)

**Comment by rafaelsteil** [comment]
Looks fine for me. I probably added it originally for utilities purposes, but if it's not used anywhere in the project, it's ok to remove. 

By the way, the time I wrote libcgi I was using php for web programming, and I wanted to have some of the facilities in C as well, so that's why there are some functions like `explode()`, `stripslashes()`, `trim()` etc.

**Comment by LeSpocky** [comment]
I noticed those functions, but this is topic for another ticket. I came across the md5 stuff because I'm reviewing the autoconf/automake stuff and this can be removed from configure and makefile then.

**Comment by DFrostByte** [comment]
I agree that there's no reason to provide MD5 as part of this library.

**Comment by apfohl** [comment]
What's the state of this issue? There is still MD5 related code inside the repository. Can it just be removed?

**Comment by LeSpocky** [comment]
The md5 related code was marked deprecated with b4a284c5ff08e93be3b496361a66051f576e26b6 and will be removed with API v2.
